### ISSUE TITLE
Infinity wires burning when connected to UIV/UEV dynamo hatch

### ISSUE THREAD

**Issue by Sphyix** [GTNewHorizons/GT-New-Horizons-Modpack#8321]
#### Which modpack version are you using?
2.1.1.0

#
#### If in multiplayer; On which server does this happen? 
delta

#
#### What did you try to do, and what did you expect to happen?
Tried making a new cable run with 1x infinity wire from an UIV dynamo hatch on a PSS

#
#### What happened instead? (Attach screenshots if needed)
Cable burned when I connected the 2nd one
Cables and UIV dynamo were in the same chunk
I tried with an UEV dynamo that is the output of a plasma turbine, same thing happened
It seems like it's happening only with 1x and 2x infinity wire and when checking it with a tricorder I can see the cable is receiving a random amount of amps
It is not happening with 4x infinity wire, but if I connect:
UIV dynamo -> 4x Infinity -> 1x Infinity
then the last 1x infinity has the random amount of amps again but the 4x does not
![javaw_4ABo8RTvxp](https://user-images.githubusercontent.com/17811922/127749807-b1859ab3-0975-466a-8c13-af2e4830350f.png)

I tested only UIV and UEV dynamo cause this is what I have in my base

#
#### What is your GTNH Discord username?
Pandoro#4811

**Comment by Sphyix** [comment]
It fixed itself after a server reboot...

**Comment by botn365** [comment]
can you send exect wire layout + things it that are atached to it with some coordinates

**Comment by Sphyix** [comment]
> can you send exect wire layout + things it that are atached to it with some coordinates

I can't reproduce it in any way
I'll share the setup that I had when it happened, I made the same setup and everything is fine now.
Cables are crossing 1 chunk
![javaw_IWwz6LSY4I](https://user-images.githubusercontent.com/17811922/127773263-b9a3aa7e-b5a1-4f97-9d94-546f8565677b.png)
![javaw_H42mfn3Grh](https://user-images.githubusercontent.com/17811922/127773276-9ce1b33c-f4bf-41a3-b13a-3781bf43143f.png)

This is what was happening yesterday
https://imgur.com/a/WhiQjcw

What I did:
Setup was off for a while, I disconnected from delta and went on eta to test other stuff with colen, I reconnected 15 minutes later, disconnected 1 cable, all the others burned
My base is entirely chunk loaded with passive anchors

Coordinates of where it started:
1st cable is at x 0, z 25, y 59
dynamo is at x 1, z 25, y 59
machines utilizing the power:
x -10, z 25, y 57
x -8, z 25, y 57
x -8, z 26, y 57
After the bug is somehow triggered, doesn't really matter where the cable is getting connected, it's always getting random amps

**Comment by Sphyix** [comment]
I forgot to add, same cable run got used with different dynamos setups.
I first tried with a UEV 64A Dynamo Hatch, discovered a bug with the molecular transformer and switched to a 2x UIV Dynamo Hatch, then switched to a single UIV Dynamo
Bug triggered when the single UIV Dynamo was used.

**Comment by Sphyix** [comment]
Happened again, when leaving the molecular transformers to run for a while..
The whole line run disappeared and wire is reading random amount of amps again
![javaw_1tAFagqBc5](https://user-images.githubusercontent.com/17811922/127786740-f89cfff7-ec9b-4137-9239-a5773997d4d9.png)

**Comment by botn365** [comment]
now that i notice it the cable that is stil behind the chunck border does not burn
but also i can not reproduce this at all 
all i can think is thermos is the problem

**Comment by Sphyix** [comment]
It's something to do with thermos, but thing is it's not happening on eta.
![javaw_zE16ezwzvo](https://user-images.githubusercontent.com/17811922/127884632-ca3c35aa-7b59-418b-a95f-034371bb89eb.png)
Eta doesn't have the enet change, delta does.
It's probably a mix of the 2 not working well together

I'll test on a dedicated server without thermos if I can reproduce this

**Comment by Sphyix** [comment]
> now that i notice it the cable that is stil behind the chunck border does not burn
> but also i can not reproduce this at all
> all i can think is thermos is the problem

the cable behind the chunk border burns aswell, I placed it back after they all burned to check amperage on it

**Comment by Sphyix** [comment]
I think this is an integer overflow...
I tried checking on the cable by spamming right click with the tricorder
![A8K0570xLl](https://user-images.githubusercontent.com/17811922/127912966-35512d3f-b974-4e42-8863-0fa786e02d11.gif)

Btw yes, it happened again after the reboot, and this time only 1x and 8x cables are bugged

**Comment by boubou19** [comment]
@botn365 you can replicate easily: place a dynamo hatch (unpowered) and connect an infinity wire to it. You'll see the amp overflow. It's unclear yet how you can trigger it with another cable, but that's a major bug if you can use another type of cable to lower the cost of the bug.

**Comment by Sphyix** [comment]
I managed to reproduce it on plain forge, testing with 2.1.1.1a

> @botn365 you can replicate easily: place a dynamo hatch (unpowered) and connect an infinity wire to it. You'll see the amp overflow. It's unclear yet how you can trigger it with another cable, but that's a major bug if you can use another type of cable to lower the cost of the bug.

This is actually how to reproduce it, boubs tested on delta where bug was already been triggered by me.
Setup a PSS with 1 UEV Dynamo hatch, connect 1x infinity wires to molecular transformers in another nearby chunk, wait a bit.
At some point the amps of the cable will overflow (i waited around 2 hours, with molecular transformers working)
Cables were working fine when I came back after the 2 hours wait, I then disconnected one of the cables from the next one and reconnected it, and they burned.
At that point bug was triggered and any new cable placed in front or connected to a generator were giving billion of amps.
